plugins/node/opentelemetry-instrumentation-graphql/package.json

```
@@ -46,7 +46,7 @@
   "devDependencies": {
     "@opentelemetry/api": "^1.3.0",
     "@opentelemetry/sdk-trace-base": "^1.8.0",
-    "@opentelemetry/semantic-conventions": "^1.3.1",
+    "@opentelemetry/semantic-conventions": "^1.22.1",
```

This should be:

```suggestion
    "@opentelemetry/semantic-conventions": "^1.22.0",
```

and then package-lock updated. There isn't a 1.22.1 release.

---

done!